server/tm_cmds.go

```diff
@@ -68,7 +67,7 @@ func ShowValidatorCmd(ctx *Context) *cobra.Command {
 			return nil
 		},
 	}
-	cmd.Flags().Bool(client.FlagJson, false, "get machine parseable output")
+	cmd.Flags().BoolP(flagMachineReadable, "m", false, "get machine parseable output")
```

Comment:
Rationale:

1. `--output` is not handled automatically as it is not a persistent flag for tendermint sub-commands
2. `--machine-parseable` is semantically quite correct as well. Added `-m` shorthand as it's pretty conventional across UNIX common cli tools.

Reply:
Not a huge fan of the `--machine-parsable` flag. I think we should just go with `--output` and define it on the cosmos side if we have to. Are there any downsides to that option?